source/imaer-gml/src/main/java/nl/overheid/aerius/gml/GMLWriter.java

```diff
@@ -202,23 +203,33 @@ private void setGMLFeatureOnObject(final InternalGMLWriter writer, final Object
     }
   }
 
+  /**
+   * @deprecated use the write method with theme parameter
+   */
+  @Deprecated
+  public File writeToFile(final File dir, final IsScenario scenario, final MetaDataInput metaData, final int fileId,
+      final Optional<String> fileNamePart, final Optional<Date> fileNameDatePart) throws AeriusException {
+    return writeToFile(dir, Theme.WNB, scenario, metaData, fileId, fileNamePart, fileNameDatePart);
+  }
+
   /**
    * Build GML and write it directly to file.
    * The file will be generated in the supplied directory.
    *
    * @param dir The directory to write the file to.
+   * @param theme theme this gml is generated for
    * @param scenario Scenario containing all scenario related data
    * @param metaData The metadata to write for this file.
    * @param fileId The ID that should be reflected in the filename.
    * @param fileNamePart The optional filename that will be incorporated in the resulting file's name.
    * @return The file generated.
    * @throws AeriusException When exception occurred generating the GML.
    */
-  public File writeToFile(final File dir, final IsScenario scenario, final MetaDataInput metaData, final int fileId,
+  public File writeToFile(final File dir, final Theme theme, final IsScenario scenario, final MetaDataInput metaData, final int fileId,
```

Comment:
Not sure if you considered it, but `theme` could be part of `MetaDataInput` instead to reduce the number of arguments required?